### ISSUE TITLE
[1.11.2 - 2.6.0.54] Wrongly registered to mod 'Minecraft'.

### ISSUE THREAD

**Issue by Trinsec** [modmuss50/Fluxed-Redstone#29]
No wonder I couldn't get any results when doing @fluxed in the JEI, because it's registered as being a part of the mod Minecraft instead of 'Fluxed Redstone'. I think something got messed up there. Took me a bit to find the items in the JEI.

**Comment by modmuss50** [comment]
Humm, that's strange. It could be to do with being written in a different lanaguage. I will take a look.

**Comment by Seegras** [comment]
I tried to rebuild, and some dependencies have changed maven repos. These are the ones the respective dependency can be found: 

http://dvs1.progwml6.com/files/maven/mezz/jei/jei_1.11/4.0.4.199/
http://mvn.rx14.co.uk/repo/mcp/mobius/waila/Hwyla/1.8.4-B16_1.11/
http://maven.tterrag.com/index.php?dir=mcjty%2Ftheoneprobe%2FTheOneProbe%2F1.11-1.3.3-46

Also, shadowJar is too old (for my gradle version), setting this to: 
classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.3'
fixed this and made it build

Then I put in an mcmod.info into resources/main, which was picked up automatically. nice.

I now tried to find the bug; but I'm giving up for now. Still, these informations might be useful for somebody wanting to rebuild it.
